public/js/helpers/humanize_helper.js

```diff
@@ -138,25 +138,31 @@ var humanize = {
       }
       return result
     }
+    var pad = function (x) {
+      if (x < 10) {
+        return `0${x}`
```

Comment:
I suspect padding with a space instead of a zero would look better, at least for the first number that doesn't even need padding since it's on the left.